services/frontend-v3/src/App.jsx

```diff
@@ -76,6 +82,59 @@ const App = () => {
             path="/secured"
             render={({ location }) => <Secured location={location} />}
           />
+          <Route
+            path="/about"
+            render={(routeProps) => {
+              const { location, match, staticContext } = routeProps;
+              return (
+                <About
+                  location={location}
+                  match={match}
+                  staticContext={staticContext}
+                />
+              );
+            }}
```

Comment:
About doesn't need all of these props, this could be simplified to
```suggestion
            render={() => <About /> }
```

and if you would just specify the type, as I mentioned in the comments below, it would rather be something like
```suggestion
            render={() => <About type="about" /> }
```

and you would do that for every route